Sources/EduIDExpansion/Flows/CreateEduID/ViewModels/CreatePincodeAndBiometricAccessViewModel.swift

```diff
@@ -84,37 +78,47 @@ final class CreatePincodeAndBiometricAccessViewModel: NSObject {
         Task {
             do{
                 let enrolment = try await TiqrControllerAPI.startEnrollmentWithRequestBuilder()
-                    .addHeader(name: Constants.Headers.authorization, value: keychain.getString(for: Constants.KeyChain.accessToken))
+                    .addHeader(name: Constants.Headers.authorization, value: keychain.getString(for: Constants.KeyChain.accessToken) ?? "")
                     .execute()
                     .body
                 
                 ServiceContainer.sharedInstance().challengeService.startChallenge(fromScanResult: enrolment.url ?? "") { [weak self] type, object, error in
                     guard let self else { return }
+                    self.secondEnteredPin.removeLast(2)
```

Comment:
Why is this required?

Reply:
the previous developer implemented the same logic for both SMS verification which uses 6 digits and for the pin verification which requires 4 digits, I noticed rewriting the entire thing need a bit more time and focus, since the project already is delayed for so long, I thought I'll rewrite once there is a better chance for it. so right now I just drop the last 2 digits so I can save a 4 digit pin code.